R/closed_curly_linter.R

```diff
@@ -37,7 +37,9 @@ closed_curly_linter <- function(allow_single_line = FALSE) {
              content_after <- unname(substr(line, parsed$col1 + 1L, nchar(line)))
              content_before <- unname(substr(line, 1, parsed$col1 - 1L))
 
-             double_curly <- rex::re_matches(content_after, rex::rex(start, "}")) || rex::re_matches(content_before, rex::rex("}", end))
+             double_curly <-
+               rex::re_matches(content_after, rex::rex(start, "}")) ||
```

Comment:
Not quite sure, but I think the default tidyverse style doesn't encourace trailing `<-`.
Would one line break after `||` be sufficient for the 120-char limit?

Reply:
I don't see anything in the style guide:

 - long lines: https://style.tidyverse.org/syntax.html#long-lines
 - assignment: https://style.tidyverse.org/syntax.html#assignment-1
 - long lines again: https://style.tidyverse.org/functions.html#long-lines-1

Locally I have `xml2`, `dplyr`, and `roxygen2`, only `roxygen2` has one line ending in `<-`:

```
grep -Er "<-$" dplyr/R | wc -l
```

So at a glance it looks avoided; will change.